gui/slick/views/home.mako

```diff
@@ -781,12 +776,13 @@ $(document).ready(function(){
             <span class="visible-print-inline">${download_stat}</span>
         </td>
 
-        <% show_size = sickbeard.helpers.get_size(curShow._location) %>
-        <td align="center" data-show-size="${show_size}">${sickbeard.helpers.pretty_filesize(show_size)}</td>
+        ## <% show_size = sickbeard.helpers.get_size(curShow._location) %>
+        ## <td align="center" data-show-size="${show_size}">${sickbeard.helpers.pretty_filesize(show_size)}</td>
+        <td align="center" data-show-size="0">Disabled!</td>
```

Comment:
I wasn't sure where the hiding was and I didn't know if we'd need to rip out half of the javascript you added. If there's an easier way to do it let me know.